src/main/groovy/wooga/gradle/snyk/SnykConventions.groovy

```diff
@@ -32,8 +32,14 @@ class SnykConventions {
             "snyk"
     )
 
+    static final PropertyLookup snykVersion = new PropertyLookup(
+            ["SNYK_VERSION", "SNYK_SNYK_VERSION"],
+            ["snyk.snykVersion", "snyk.version"],
+            "v1.840.0"
```

Comment:
The install task would fail then. It always requires a version. But I believe this shouldn't ever be empty as v1.840.0 is the default value, right?

Reply:
We could put "latest" as the default value, so it would always download the latest version by default, but I believe its way better for us to pin our version and focus on supporting the pinned version.

Reply:
I would provide a use latest way. I don't want to have yet another reason to open 50 to 100 PRs. At least we can set it on agent level with an environment variable. Let's implement latest later :)

Reply:
We don't have to update it every time. The user can set it to "latest" himself through the extension if he wants to.